frontend/src/hooks/useGetFilters.tsx

```diff
@@ -6,56 +6,96 @@ import {
   RegionName,
 } from '@/common';
 import type { Region } from '@/common/enums/region.enum';
-import type { FunctionType, Location } from '@/pages/dashboard';
+import type { FunctionType, Location, SectionType } from '@/pages/dashboard';
 import { DashboardFilterNames } from '@/pages/dashboard';
 import { useEffect, useState } from 'react';
 import { useAxios } from './useAxios';
+import { useRole } from './useRole';
+import { Route } from '@/providers';
+import type { FireCentre } from '@/common/enums/firecentre.enum';
+import { FireCentreName } from '@/common/enums/firecentre.enum';
 
 export const useGetFilters = () => {
   const [locations, setLocations] = useState<Location[]>([]);
   const [regions, setRegions] = useState<Region[]>([]);
+  const [sections, setSections] = useState<SectionType[]>([]);
+  const [fireCentre, setFireCentre] = useState<FireCentre[]>([]);
   const [functions, setFunctions] = useState<FunctionType[]>([]);
+  const { route } = useRole();
   const { AxiosPrivate } = useAxios();
 
   const sortRegion = (reg: Region[]) => {
     return reg.sort((a: Region, b: Region) => a.localeCompare(b));
   };
 
+  const sortFireCentre = (centre: FireCentre[]) => {
+    return centre.sort((a: FireCentre, b: FireCentre) => a.localeCompare(b));
```

Comment:
Would this fit better in the backend?

Reply:
I don't think so - we are returning locations list in a proper order, and then just re-ordering the associated fire centres for displaying in the drop down. If we order by fire centre then locations will be out of order.